chrome-stable/entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#! bin/sh
+socat tcp-listen:9222,reuseaddr,fork tcp:localhost:9223 &
+/usr/bin/google-chrome --disable-gpu --headless --disable-dev-shm-usage --remote-debugging-port=9223 --user-data-dir=/data
```

Comment:
Can we make this script support taking additional arguments, i.e. adding $@ to the end of this line?

Reply:
I will make another PR.